Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -56,3 +56,23 @@
 DigiToHybridRawRepack = cms.Sequence( DigiToHybridRawRepackTask )
 DigiToVirginRawRepack = cms.Sequence( DigiToVirginRawRepackTask )
 DigiToSplitRawRepack = cms.Sequence( DigiToRawRepackTask, DigiToVirginRawRepackTask )
+
+from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
+#siStripDigis.ProductLabel = "rawDataRepacker"
```

Comment:
IIUC, `approximatedClusters` can disambiguate/allow to pick a correct value here

Reply:
Sorry, it's still not clear for me.  I just pushed a temporary solution using customise_commands such that at least 161 and 140.58 should run.  If you can recommend how to do this properly, I'd be much obliged.